steps/terraform-precheck.yaml

```diff
@@ -0,0 +1,65 @@
+parameters:
+  - name: adolibrary
```

Comment:
why does this need to be a parameter?

Reply:
now removed and hardcoded. couldn't get working as a variable using $(var) and ${{ variables.name }} formats.